tests/capture/test_base_interpreter.py

```diff
@@ -543,3 +543,59 @@ def circuit(y):
         assert qfunc_jaxpr.eqns[1].primitive == qml.RX._primitive  # eqn 0 is mul
         assert qfunc_jaxpr.eqns[2].primitive == qml.Z._primitive
         assert qfunc_jaxpr.eqns[3].primitive == qml.ops.SProd._primitive
+
+
+class TestDynamicShapes:
+    """Test that our interpreters can handle dynamic array creation."""
+
+    @pytest.mark.parametrize("reinterpret", (True, False))
+    def test_creating_ones(self, reinterpret):
+        """Test that broadcast_in_dim can be executed with PlxprInterpreter."""
+        jax.config.update("jax_dynamic_shapes", True)
```

Comment:
Sorry to sound like a broken record, but this is another example for needing a global fixture for managing enabling/disabling dynamic shapes 😅

Reply:
using the fixture now.